src/components/TopBar/ProjectPickerButton.jsx

```diff
@@ -1,16 +1,35 @@
+import classnames from 'classnames';
 import React from 'react';
 import {t} from 'i18next';
+import PropTypes from 'prop-types';
 
-export default function ProjectPickerButton() {
+export default function ProjectPickerButton({isSaveIndicatorShown}) {
   return (
-    <span>
-      {t('top-bar.load-project')}
-      {' '}
-      <span className="u__icon top-bar__drop-down-button">
-        &#xf0d7;
+    <div>
+      <span
+        className={
+          classnames({'top-bar__project-button-hidden': isSaveIndicatorShown})
+        }
+      >
+        {t('top-bar.load-project')}
+        {' '}
+        <span className="u__icon top-bar__drop-down-button">
+          &#xf0d7;
+        </span>
       </span>
-    </span>
+      <span
+        className={
+          classnames(
+            'top-bar__project-saved',
+            {'top-bar__project-saved-shown': isSaveIndicatorShown})
+        }
+      >
+        {t('top-bar.project-saved')}
+      </span>
+    </div>
   );
 }
 
-ProjectPickerButton.propTypes = {};
+ProjectPickerButton.propTypes = {
+  isSaveIndicatorShown: PropTypes.bool.isRequired,
```

Comment:
Weakly held opinion, but `shouldShowSavedIndicator` would read more naturally to me here.